__tests__/app.test.js

```diff
@@ -160,6 +161,25 @@ describe("GET /api/articles", () => {
         });
       });
   });
+  test("200 the number of articles associated with given topic has the correct length", () => {
+    return request(app)
+      .get("/api/articles?topic=mitch")
+      .expect(200)
+      .then((res) => {
+        const { article } = res.body;
+
+        expect(article.length === 12).toBe(true);
+      });
+  });
+  test("400: returns error message when sending a get request to a topic that does not exist", () => {
+    return request(app)
+      .get("/api/articles?topic=not_a_topic")
```

Comment:
I guess not_a_topic would probably never be a topic and in this case you could build it how you would want to. I would argue that whatever we might consider to be an 'invalid topic' could be seen as, valid but just doesn't exist yet. Perhaps testing for a 404 would be better ?